test/support/test_config.h

```diff
@@ -120,6 +120,54 @@
+
+#define _PSTL_TEST_COMPLEX_ACOS_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ACOS_MINUS_ZERO_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ACOS_NAN_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_ACOSH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ACOSH_NAN_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_ASINH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ASINH_NAN_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ASINH_NAN_MINUS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_SIN_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SIN_MINUS_ZERO_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SIN_NAN_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_ATANH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ATANH_MINUS_ZERO_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_ATANH_NAN_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_COSH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_MINUS_ZERO_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_MINUS_INF_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_MINUS_ZERO_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_INF_FINITE_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_NAN_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COSH_MINUS_INF_MINUS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_ZERO_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_INF_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_ZERO_MINUS_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_INF_LESS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_NAN_MINUS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_SINH_MINUS_INF_MINUS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_TANH_MINUS_INF_NAN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_TANH_INF_INF_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_TANH_NAN_MINUS_ZERO_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_SIN_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_COS_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_LOG10_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+
+#define _PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN
+#define _PSTL_TEST_COMPLEX_TIMES_COMPLEX_BROKEN _PSTL_TEST_COMPLEX_OP_BROKEN

```

Comment:
Defining a separate macro for each test/operation makes no sense to me. Why is just one generic macro not enough?

Reply:
This is our common approach in tests to have some separate macro for each broken case.
But or course I am ready to use `_PSTL_TEST_COMPLEX_OP_BROKEN` instead of them.